src/Support/SupportData.php

```diff
@@ -620,11 +652,98 @@ static function ( $url ) {
 					'urls'          => [],
 				];
 			}
+
+			// If no specific URL requested.
+			$query_args = [
+				'post_type'      => AMP_Validated_URL_Post_Type::POST_TYPE_SLUG,
+				'posts_per_page' => 100,
+				'post_status'    => 'publish',
+				'meta_key'       => AMP_Validated_URL_Post_Type::VALIDATED_ENVIRONMENT_POST_META_KEY,
+				'meta_value'     => maybe_serialize( AMP_Validated_URL_Post_Type::get_validated_environment() ), // phpcs:ignore WordPress.DB.SlowDBQuery.slow_db_query_meta_value
+			];
+
+			$query               = new WP_Query( $query_args );
+			$amp_validated_posts = $query->get_posts();
+
+			foreach ( $amp_validated_posts as $amp_validated_post ) {
+				$validation_data = $this->process_raw_post_errors( $amp_validated_post );
+
+				if (
+					! empty( $validation_data['errors'] )
+					&&
+					! empty( $validation_data['error_sources'] )
+					&&
+					! empty( $validation_data['url'] )
+				) {
+					$errors             = array_merge( $errors, $validation_data['errors'] );
+					$error_sources      = array_merge( $error_sources, $validation_data['error_sources'] );
+					$amp_invalid_urls[] = $validation_data['url'];
+				}
+			}
+		} else { // If we have specific URL requested.
+
+			foreach ( $this->urls as $url ) {
+
+				// Check request URL have validated data if not then validate URL.
+				$amp_validated_post_id = post_exists( $url, '', '', AMP_Validated_URL_Post_Type::POST_TYPE_SLUG );
+
+				if ( empty( $amp_validated_post_id ) ) {
+					$validity              = AMP_Validation_Manager::validate_url_and_store( $url );
+					$amp_validated_post_id = $validity['post_id'];
```

Comment:
@dhaval-parekh As I mentioned in one of the comments above, the "url" attribute made of post title ends up with something like `"https://Test%20embeds"` here, which then throws a Fatal error:

`Fatal error: Uncaught Error: Cannot use object of type WP_Error as array in /var/www/html/wp-content/plugins/amp-wp/src/Support/SupportData.php:692 Stack trace: #0 /var/www/html/wp-content/plugins/amp-wp/src/Support/SupportData.php(195): AmpProject\AmpWP\Support\SupportData->get_amp_urls() #1 /var/www/html/wp-content/plugins/amp-wp/src/Admin/SupportScreen.php(177): AmpProject\AmpWP\Support\SupportData->get_data() #2 /var/www/html/wp-includes/class-wp-hook.php(303): AmpProject\AmpWP\Admin\SupportScreen->enqueue_assets('amp_page_amp-su...') #3 /var/www/html/wp-includes/class-wp-hook.php(327): WP_Hook->apply_filters(NULL, Array) #4 /var/www/html/wp-includes/plugin.php(470): WP_Hook->do_action(Array) #5 /var/www/html/wp-admin/admin-header.php(102): do_action('admin_enqueue_s...', 'amp_page_amp-su...') #6 /var/www/html/wp-admin/admin.php(239): require_once('/var/www/html/w...') #7 {main} thrown in /var/www/html/wp-content/plugins/amp-wp/src/Support/SupportData.php on line 692`

Reply:
I have added a check for it to handle that scenario.